react-big-calendar/index.d.ts

```diff
@@ -38,6 +38,8 @@ declare module 'react-big-calendar' {
         messages?: Object;
         timeslots?: number;
         defaultView?: string;
+        className?: string;
+        elementProps?: React.HTMLAttributes<HTMLElement>;
```

Comment:
that's cool but I cannot see the commit though, have you made push to your branch?